### ISSUE TITLE
Mesa regression 11.0.2 GL_INVALID_OPERATION

### ISSUE THREAD

**Issue by Earnestly** [Cloudef/orbment#130]
Spawning any new surface results in the following errors:
```
(ERROR) wlc: gles2: function shm_attach at line 686: gl.api.glTexImage2D(0x0DE1, 0, gl_format, pitch, buffer->size.h, 0, gl_format, gl_pixel_type, data) == GL_INVALID_OPERATION
```

This is a regression in mesa 11.0.2 https://bugs.freedesktop.org/show_bug.cgi?id=92265

**Comment by Earnestly** [comment]
This is now fixed in mesa 11.0.3

**Comment by Cloudef** [comment]
Thanks
